templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,41 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List
```

Comment:
The `<h1>` tag is not properly closed on the same line. Consider moving the closing tag `</h1>` to the same line as the opening tag for better readability: `<h1>Manufacturer List</h1>`.